src/main/java/com/google/step/YOUR_PROJECT_NAME_HERE/data/Card.java

```diff
@@ -0,0 +1,54 @@
+package com.google.step.YOUR_PROJECT_NAME_HERE.data;
+
+public final class Card {
+  private String title;
+  private String code;
+  private String link;
+  private String description;
+
+  public Card() {
+    this.title = "";
+    this.code = "";
+    this.link = "";
+    this.description = "";
+  }
```

Comment:
this looks strange to me. i think it should be an default constructor

Reply:
ah, good point, I set the variables to initialize as empty instead, the default constructor shouldn't need to be explicitly generated